test/jdk/java/io/File/GetCanonicalPath.java

```diff
@@ -122,4 +125,96 @@ void driveLetter() throws IOException {
         String path = new File("c:/").getCanonicalPath();
         assertFalse(path.length() > 3, "Drive letter incorrectly represented");
     }
+
+    private static Path createPath(String pathname) throws IOException {
```

Comment:
This method creates a File, returning a Path to the file, so maybe better method name needed here.

Reply:
Changed to `createFile` and added a one line comment in 796a634.